src/main/java/org/spongepowered/api/item/inventory/entity/UserInventory.java

```diff
@@ -42,32 +42,32 @@
      *
      * @return The main inventory
      */
-    MainPlayerInventory getMain();
+    MainPlayerInventory getPrimary();
```

Comment:
What is wrong with `getMain` here?
If you change this then should the transformation `PLAYER_MAIN_HOTBAR_FIRST` (https://github.com/SpongePowered/SpongeAPI/blob/bleeding/src/main/java/org/spongepowered/api/item/inventory/InventoryTransformations.java#L42) be renamed to `PLAYER_PRIMARY_HOTBAR_FIRST`?

Reply:
`getPrimary` is a nicer name, in my opinion.

Reply:
Maybe also renaming the class then? `PrimaryPlayerInventory`?